Automated response actions for rules (#3546)
elastic/security-docs: docs/management/admin/host-isolation-ov.asciidoc

[chapter, role="xpack"]
= Host isolation

> :frontmatter-description: Host isolation allows you to isolate hosts from your network, blocking communication with other hosts on your network until you release the host. 

Reviewer: ```suggestion
:frontmatter-description: Host isolation allows you to cut off a host's network access until you release it.
```

. Press *Return*.
====

> .Automatically isolate a host through a rule's endpoint response action

Reviewer: ```suggestion
.Automatically isolate a host using a rule's endpoint response action
```

Author: I went back and forth on this, so I'm glad you left this suggestion :)

[%collapsible]
====
NOTE: The host isolation endpoint response action is an https://www.elastic.co/pricing[Enterprise subscription] feature. 

> IMPORTANT: Automatic host isolation from a detection rule execution could result in unintended consequences, such as disrupting legitimate user activities or blocking critical business processes. Be aware of this as you're configuring rules. 

Author: ```suggestion
IMPORTANT: Be aware that automatic host isolation can result in unintended consequences, such as disrupting legitimate user activities or blocking critical business processes.
```
